requirements.tex

```diff
@@ -3,7 +3,12 @@ \section{Requirements} \label{sec:reqs}
 There are several different stakeholders of the \gls{LSST} \gls{DM} software products, each of whom may have different requirements for releases.
 In this section we give an overview of their respective requirements on  \gls{DM} software product releases.
 
-To be kept in mind that the content of a release, must be well understood and documented in the corresponding release note.
+It is important to identify the correct release policy during contraction phase, in order to have it consolidated when operations starts.
```

Comment:
It is important to identify the correct release policy during the construction phase, in order to have it consolidated when operations starts.